docs/website/docs/dlt-ecosystem/destinations/duckdb.md

```diff
@@ -77,7 +77,45 @@ to disable tz adjustments.
 :::
 
 ## Supported column hints
-`duckdb` may create unique indexes for all columns with `unique` hints, but this behavior **is disabled by default** because it slows the loading down significantly.
+
+`duckdb` can create unique indexes for columns with `unique` hints. However, **this feature is disabled by default** as it can significantly slow down data loading.
+
+## Supported column flags
```

Comment:
those are still called hints. I'd move this section under
## Data loading

and replace this header with 
### Data types

(it is used in several other destinations)

please apply this to other docs

Reply:
Done!

**Question**:  What about this section? 
```
## Supported column hints

`duckdb` can create unique indexes for columns with `unique` hints. However, **this feature is disabled by default** as it can significantly slow down data loading.
```
It was there before this PR. Should we keep it as it is, or move it under **Data Loading** as well?